composer.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "knplabs/knp-menu-bundle",
+    "minimum-stability": "beta",
```

Comment:
Do we need this line? Or could this be added to the CI only?

Reply:
Probably not, I had that (and the symfony extra) to make sure that tests pass.  How is that added to CI?

Reply:
One way could be like this. https://github.com/tattali/CalendarBundle/pull/64/files#diff-f47fc96cfa328e123eaec41fe0ef2d23567991431dbdeef87da6c024dd103c08R61

Or a seperate CI-File for dev